.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # ignore generated files
 *.elc
+.DS_Store
```

Comment:
I don't think we have to add `.DS_Store` not everybody used a mac.
I believe that if you use one, you could add it to `~/.config/git/ignore` to ignore it from all your projects.

Reply:
Thanks for that tip!